cogs/greetings.py

```diff
@@ -1,27 +1,40 @@
 import discord
 from discord.ext import commands
 
-#Bot sends out a greeting to the server when a new member joins the server
+# Sends a greeting message that requires a new user to react to the message in order to receive a role to access the server
 class Greetings(commands.Cog):
-
-    def __init__(self,client):
+    def __init__(self, client):
         self.client = client
-        self._last_member = None
 
     @commands.Cog.listener()
     async def on_member_join(self, member):
-        #Sends the embedded message to the channel that is the welcome page
-        channel = member.guild.system_channel
+        channel = self.client.get_channel(1011049173442895902)  # Replace with the actual ID of your welcome text channel
```

Comment:
hardcoding channel ID is probably not a good idea. we should make a configuration file that holds this value.